states/kentucky/elections.toml

```diff
@@ -22,3 +22,28 @@ in_person_by = 2020-06-23
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+online_by = 2020-10-05T16:00:00
+received_by = 2020-10-05T16:00:00
+in_person_by = 2020-10-05T16:00:00
+sources = ["https://elect.ky.gov/Resources/Documents/2020%20Election%20Calendar%20with%20Primary%20Postponement.pdf"]
+
+[20201103.absentee.application]
+received_by = 2020-10-27
```

Comment:
The popup on https://vrsws.sos.ky.gov/abrweb/ says online_by 10/9 midnight.

Reply:
PDF says:

>Last day to apply for mail-in absentee ballot (not later than close of business   7   days   before   election).   Applications,   including   the   declaration accompanying a federal write-in absentee ballot, must be received by this day.

KY's site is a mess. I guess we should error to the earlier date. Agree?

Reply:
Ya, I think the earlier is better. Otherwise we risk folks not getting their ballots.